backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/externallyreferenced/ExternallyReferencedCandidateRepository.java

```diff
@@ -316,13 +317,29 @@ private void storeInvoiceDetailItems(final InvoiceCandidateId invoiceCandidateId
 
 	private I_C_Invoice_Detail createI_C_InvoiceDetail(final InvoiceCandidateId invoiceCandidateId, final InvoiceDetailItem invoiceDetailItem)
 	{
+		
 		final I_C_Invoice_Detail invoiceDetailEntity = newInstance(I_C_Invoice_Detail.class);
 
 		invoiceDetailEntity.setC_Invoice_Candidate_ID(invoiceCandidateId.getRepoId());
```

Comment:
please also set invoiceDetailEntity.AD_Org_ID here

Reply:
you :+1: but you didn't do it :-P

Reply:
already there